boa/src/builtins/symbol/tests.rs

```diff
@@ -1,13 +1,5 @@
-use super::*;
 use crate::{exec::Interpreter, forward, forward_val, realm::Realm};
 
-#[test]
-fn check_symbol_constructor_is_function() {
-    let global = Value::new_object(None);
-    let symbol_constructor = Symbol::create(&global);
-    assert_eq!(symbol_constructor.is_function(), true);
-}
-
```

Comment:
Is there a reason you've removed these tests?

Reply:
> Is there a reason you've removed these tests?

It's because `create` function was removed.

Reply:
Do we need to care that init returns a function type like before?

Reply:
Not really, if init was to not return a function for example symbol all the other tests calling symbol would fail so that scenario is covered.